python/nano/src/bigdl/nano/deps/neural_compressor/tensorflow/metric.py

```diff
@@ -21,9 +21,9 @@ class TensorflowINCMetric(BaseINCMetric):
     def stack(self, preds, labels):
 
         # calculate accuracy
-        preds = tf.stack(preds)
         labels = tf.stack(labels)
-        return preds, labels
+        preds = tf.stack(preds)
+        return labels, preds
```

Comment:
I think you should override the `result` function instead.
```suggestion
        preds = tf.stack(preds)
        labels = tf.stack(labels)
        return preds, labels

    def result(self):
        # calculate accuracy
        preds, labels = self.stack(self.pred_list, self.label_list)
        accuracy = self.metric(labels, preds)
        return self.to_scalar(accuracy)
```